README.md

```diff
@@ -39,3 +39,55 @@ Bounce uses a number of open source projects and APIs to work properly:
 * [LocationIQ API](https://locationiq.com/) - used to get reverse geocoding results for markers
 * [Mapbox API](https://www.mapbox.com) - used to provide forward-geocoding results and mapping
 * [PostgreSQL](https://www.postgresql.org/) - SQL database for scalability, modularity, and geoquerying support
+
+# Setup
+
+## Docker
+
+0. Install [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+
+1.  Make `docker_entrypoint.sh` executable
+    ```bash
+    # Linux
+    chmod +x docker_entrypoint.sh
```

Comment:
You can definitely do this prior to committing, so that it's executable by default.

Reply:
Thanks! I assumed this was a security measure from git, so I didn't knew it could be changed.

I used `git add docker_entrypoint.sh --chmod=+x` [Reference](https://www.git-scm.com/docs/git-add#Documentation/git-add.txt---chmod-x)